www/config.yml

```diff
@@ -4,6 +4,12 @@ archives_url: /archives/
 author: A PyLady
 base_url: /
 chapters:
+- email: congo@pyladies.com
+  image: PyLadiesCongo.png
```

Comment:
Where it this image? I do not see it checked into the pull request? See: https://github.com/pyladies/pyladies/pull/438/files.